format_spec.md

```diff
@@ -0,0 +1,289 @@
+
+##### `name`
+
+The human-readable name of the license. Examples include: "MIT" and "LGPLv3".
+
+##### `uri`
+
+A URL linking to the full license.
+
+##### `modpackUsage` (optional)
+
+A field describing usage of the Project in modpacks. Valid options:
+
+- `alwaysAllow` - This Project can be used in a modpack without asking for permission.
+- `requiresPermission` - Before included in a modpack, permission must be granted from the author.
+- `neverAllow` - This Project can never be used in a modpack.
+
+---
+
+### `links` (optional)
+
+This field MUST be stored as an array. It contains a collection of links relating to the Project. Each Link Object stored inside this arrow MUST contain the following fields:
+
+##### `rel`
+
+This field MUST be stored as an array OR a String. If stored as an array, it MUST contain Strings. This field contains information about what the link is. Valid standardized values are:
+
+- `homepage` - The official homepage of a Project
+- `documentation` - Documentation of a Project.
+- `wiki` - Wiki information about a Project.
+- `scm` - The source code of a Project, for example a GitHub URL.
+- `issues` - An issue tracker where bugs and feature requests can be filed.
+- `community` - A forum or other online discussion platform where users can discuss the Project.
+
+Other values MAY be used, such as `bitbucket` or `github`. However, it is RECOMMENDED to use generic terminology instead of specific names. If you are unsure whether a value COULD become standardized, please prefix it with `x-`.
+
+##### `name`
+
+A human-readable name of the link. For example, "Source Code" or "Project Website".
+
+##### `uri`
+
+The link, to be viewed in a web browser.
+
+---
+
+### `successor` and `predecessor` (optional)
+
+If a newer version of a Project has been released and it does not have the same ID as the previous version, the `successor` and `predecessor` fields may be set for the appropriate Projects. This fields MUST contain the ID of the predecessor or successor project.
+
+---
+
+## Version Information
+
+Projects MAY contain a `versions` array, which is made up of Version Objects. If you are writing an API that uses the MCIP format, this field MAY be separated into a separate API call in order to save bandwidth. The `versions` field MAY also be paginated in API calls.
+
+## Version Objects
+
+Version Objects contain fields relating to a certain version of a Project. Below are all the fields that a Version Object may contain.
+
+### `id`
+
+The ID of this version. There is no required format for how to format a version ID, other than that they CANNOT include spaces.
+
+---
+
+### `name`
+
+The human-readable name of this version. It is RECOMMENDED to follow Semantic Versioning, but it is not required. Spaces MAY be used in this name.
+
+---
+
+### `semver` (optional)
+
+An optional Semantic Versioning compatible version number.
```

Comment:
Does it have to be related to the actual mod version or is this just for better tracking compatibility? Can it change later if an incompatibility is found?